addons/website_livechat/static/tests/mock_server/mock_models/discuss_channel.js

```diff
@@ -29,7 +29,7 @@ export class DiscussChannel extends livechatModels.DiscussChannel {
                 const [partner] = ResPartner.browse(visitor.partner_id);
                 const [country] = ResCountry.browse(visitor.country_id);
                 channelInfo.visitor = {
-                    country: country ? { id: country.id, code: country.code } : false,
+                    country_id: country ? { id: country.id, code: country.code } : false,
```

Comment:
To be closed to python it would be best if this mock server added the country id here, and made a separate store.add to add the country data.

Reply:
Or basically create a proper to store for visitor in the mock server.